tests/run/returns.scala

```diff
@@ -0,0 +1,16 @@
+def foo(x: Int): Int =
+  if x == 1 then
+    return
```

Comment:
Should there be a test that also checks multiple indented lines? This looks like a code that should work anyway - single statement return.

Reply:
done